src/components/dev-dot-content/mdx-components/mdx-headings/mdx-headings.module.css

```diff
@@ -38,18 +38,25 @@
 
 .h1 {
 	color: var(--token-color-foreground-strong);
-	margin-bottom: 12px;
+
+	/* "Display 600" custom font metrics, outside HDS */
```

Comment:
Have used Display 500 as a base to build the "Display 600" styles for `h1` elements in MDX content.

📌 We could refactor this as an additional helper class, named like the typical HDS helper classes, and add it into our `Heading` component. On the fence about whether to do that as part of this PR, or whether to wait until we have another use case for Display 600 before codifying the abstraction.

Reply:
Update: Second use case came up, in the Tutorials Meta component. So I've added `hds-typography-display-600.css` as a temporary way to make it feel like `display-600` exists in HDS. Also added the prop option to our `Heading` component.